<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogOutputWdg</class>
 <widget class="QWidget" name="LogOutputWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>140</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutType">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="XMatLabel" name="lbLogType">
       <property name="text">
        <string>日志类型:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatComboBox" name="cmbLogTypeBind">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatComboBox" name="cmbLogTypeVal">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="XMatHLine" name="xMatHLine"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutMsg">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="XMatLabel" name="lbMsg">
       <property name="text">
        <string>消息:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatComboBox" name="cmbMsg">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatPlainTextEdit" name="ptxMsg">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XMatPlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>XMatPlainTextEdit.h</header>
  </customwidget>
  <customwidget>
   <class>XMatComboBox</class>
   <extends>QComboBox</extends>
   <header>XMatComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>XMatLabel</class>
   <extends>QLabel</extends>
   <header>XMatLabel.h</header>
  </customwidget>
  <customwidget>
   <class>XMatHLine</class>
   <extends>QWidget</extends>
   <header>XMatHLine.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
